.github/actions/linting/action.yml

```diff
@@ -9,34 +9,26 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: hashicorp/setup-terraform@v2
-      with:
-        terraform_version: ">=1.5.0"
+    - name: grept
+      run: exit 0
+      shell: bash
 
-    - name: terraform init
-      run: terraform init
+    - name: terraform fmt check
+      run: exit 0
```

Comment:
i'm curious why the run commands in this file are "exit 0" ?

Reply:
This is so @lonegunmanb can update this branch with the actual tasks. He will use the docker container approach pioneered with TFVM.